### DISCUSSION TITLE
Err SSL Key on latest Chrome

### DISCUSSION THREAD

**Discussion by its-a-me-mario-the-og** [Security-Onion-Solutions/securityonion#11814] [Unsupported Versions] [answered]
Hey SO team, chrome released an update within the past 3-7 days and after updating I can not access any SO boxes. This affects both 2.3 and 2.4 versions. The version of Chrome I am running is 119.0.6045.160, one of my coworkers who helps with the SO boxes updated his Chrome to the latest and he received the same error message. Here is a screenshot of the error message on Chrome.
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/91745779/eacb93d2-a44e-4366-8777-3f70507fbc56)
I did look up the error message and tried to a few different ways to get it to working because I thought Chrome was the problem. But this still persists after trying the different methods. Any guidance on how to get this fixed will be greatly appreciated!

Edit: It does work with Firefox and Edge, I have tried both of those browsers, as far as I know this just affects Chrome. I haven't tried it with Brave or any other browser.

**Comment by TOoSmOotH** [comment]
We are working on incorporating a work around in our next hotfix. In the meantime:

https://github.com/Security-Onion-Solutions/securityonion/discussions/11803#discussioncomment-7591602

**Reply by its-a-me-mario-the-og** [reply]
Thank you!

**Comment by dougburks** [comment, marked as answer]
We've released a hotfix:
https://blog.securityonion.net/2023/11/security-onion-2430-hotfix-20231117-now.html
